src/main/java/com/iexec/worker/tee/scone/LasService.java

```diff
@@ -0,0 +1,95 @@
+package com.iexec.worker.tee.scone;
+
+import com.iexec.common.docker.DockerRunRequest;
+import com.iexec.common.docker.DockerRunResponse;
+import com.iexec.common.docker.client.DockerClientInstance;
+import com.iexec.worker.config.WorkerConfigurationService;
+import com.iexec.worker.docker.DockerService;
+import com.iexec.worker.sgx.SgxService;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class LasService {
+    @Getter
+    private final String containerName;
+    private final String imageUri;
+    @Getter
+    private final SconeConfiguration sconeConfig;
+
+    private final WorkerConfigurationService workerConfigService;
+    private final SgxService sgxService;
+    private final DockerService dockerService;
+
+    @Getter
+    private boolean isStarted;
+
+    public LasService(String containerName,
+                      String imageUri,
+                      SconeConfiguration sconeConfig,
+                      WorkerConfigurationService workerConfigService,
+                      SgxService sgxService,
+                      DockerService dockerService) {
+        this.containerName = containerName;
+        this.imageUri = imageUri;
+        this.sconeConfig = sconeConfig;
+        this.workerConfigService = workerConfigService;
+        this.sgxService = sgxService;
+        this.dockerService = dockerService;
+    }
+
+    boolean start() {
+        if (isStarted) {
+            return true;
+        }
+
+        DockerRunRequest dockerRunRequest = DockerRunRequest.builder()
+                .containerName(containerName)
+                .imageUri(imageUri)
+                // application & post-compose enclaves will be
```

Comment:
```suggestion
                // optional pre-compose, application & post-compose enclaves will be
```

Reply:
We're talking about `pre-compute` and `post-compute`, not `pre-compose` and `post-compose`, right?
See https://github.com/iExecBlockchainComputing/iexec-worker/pull/464/commits/52e3e325bf706ceb7f2edd3c01d5fc4fadc83f22.

Reply:
Yes you are right!